rmf_fleet_adapter/src/rmf_fleet_adapter/agv/internal_FleetUpdateHandle.hpp

```diff
@@ -208,7 +208,7 @@ class FleetUpdateHandle::Implementation
     handle->_pimpl->fleet_state_pub = handle->_pimpl->node->fleet_state();
     handle->_pimpl->fleet_state_timer =
       handle->_pimpl->node->try_create_wall_timer(
-      std::chrono::seconds(1), [me = handle->weak_from_this()]()
```

Comment:
If the default value is changed, we will also need to update the comment [here] in the header. Hmmm also, not sure if we will need to change this 1s value, since the user can update it via the `fleet_state_publish_period` function.

Reply:
Ah okay! For the demo adapter, the user just needs to modify the `robot_state_update_frequency` value ([example here](https://github.com/xiyuoh/rmf_demos/blob/feature/demo_fleet_adapter/rmf_demos/config/hotel/deliveryRobot_config.yaml#L52)), it will publish according to that frequency even if the default in `FleetUpdateHandle` is 1Hz. I initially thought we should increase the default frequency, but I can undo this if needed!

Reply:
@xiyuoh the `robot_state_update_frequency` is not linked to the frequency of the published `FleetState` msgs. The former controls the rate at which the robots's position is updated to the `RobotUpdateHandle`. It does not publish any message. There is a separate timer within `FleetUpdateHandle` that retrieves the latest position/state of the robot and publishes a `FleetState` message. The periodicity of this timer is controlled by [this api](https://github.com/open-rmf/rmf_ros2/blob/ccf18ea463d979c4d23b8fba60df6812dd593b5e/rmf_fleet_adapter/include/rmf_fleet_adapter/agv/FleetUpdateHandle.hpp#L179-L183). So ideally, both rates are set to the same value.